README.md

```diff
@@ -18,6 +18,10 @@ bridge, and one to invoke calls in the backend.
 Similarly to the Desktop variant, the Go library can be statically compiled and added to an Android
 Studio / XCode project. This is not part of this repo yet.
 
+### Compatibility
+x86 (no ARM), 64 bit: Windows 7+, macOS 10.11+, Linux 
```

Comment:
pls nicer formatting (bullet points, etc)

Reply:
ping @jstrnbrg

eg x86 implies it is not ARM, can also elaborate for Linux: Ubuntu 16.04+; Fedora; other distributions via AppImage.

Reply:
@benma ups sorry. TBH I think it is quicker if you just write it the way you want it ;)